notebooks/official/model_evaluation/automl_video_classification_model_evaluation.ipynb

```diff
@@ -483,7 +426,7 @@
         "\n",
         "This tutorial uses a version of the MIT Human Motion dataset that is stored in a public Cloud Storage bucket, using a CSV index file.\n",
         "\n",
-        "Start by doing a quick peek at the data. You count the number of examples by counting the number of rows in the CSV index file  (`wc -l`) and then peek at the first few rows."
+        "Start by doing a quick peek at the data. You count the number of examples by counting the number of rows in the CSV index file  (wc -l) and then peek at the first few rows."
```

Comment:
Modify to: "Start by taking a quick peek at the data. Count the number of examples by counting the number of rows in the CSV index file (wc -l) and then peek at the first few rows."